src/Workspaces/Core/Portable/FlowAnalysis/SymbolUsageAnalysis/SymbolUsageAnalysis.AnalysisData.cs

```diff
@@ -240,6 +257,7 @@ protected virtual void DisposeCoreData()
             {
                 SymbolsWriteBuilder.Free();
                 SymbolsReadBuilder.Free();
+                _referenceTakenSymbolsBuilder.Free();
```

Comment:
It seems base.DisposeCoreData() isn't always called in FlowGraphAnalysisData, are we leaking those pooled memory?

Reply:
Good catch! Thanks, fixed now.